hw3/01-characters.js

```diff
@@ -0,0 +1,39 @@
+const url = 'https://thronesapi.com/api/v2/Characters';
+
+fetch(url)
+  .then(response => response.json())
+  .then(characters => {
+    const section = document.querySelector('section');
+
+    characters.forEach(character => {
+      const div = document.createElement('div');
+      const img = document.createElement('img');
+      const h2 = document.createElement('h2');
+      const p = document.createElement('p');
```

Comment:
Please consider using more meaningful names. For example, since the h2 will contain the character's name it should be called something along the lines of "characterName". 

Same thing for "p", which is not a meaningful name. Since it will be used for the "title", it should be called that.